model/src/test/test_periodiceffect.py

```diff
@@ -44,7 +44,7 @@ def test_periodiceffect() -> None:
     params["offset"] = 5
     pe = PeriodicEffect(**params)
     with npro.handlers.seed(rng_seed=np.random.randint(1, 600)):
-        ans2 = pe(duration=duration).value
+        ans2 = pe(duration=duration).value.value
```

Comment:
I think this has implications for our naming conventions for the `NamedTuples` returned by `RandomVariable.sample()`. For now I would suggest this:
```suggestion
        ans2 = pe(duration=duration)[0].value
```